todo_app/data/trello_items.py

```
@@ -0,0 +1,116 @@
+import os
+from requests import request, exceptions
+import itertools
+
+from todo_app.data.item import Item
+from todo_app.data.list import List
+
+trello_base_url = "https://api.trello.com" 
+api_key = os.environ.get('TRELLO_API_KEY')
+api_token = os.environ.get('TRELLO_TOKEN')
+board_id = os.environ.get('TRELLO_BOARD_ID')
+
+def trello_request(method, path, query_params = []):
+    headers = {
+        "Accept": "application/json"
+    }
+    url = f"{trello_base_url}{path}?key={api_key}&token={api_token}"
+
+    for query_param in query_params:
+        url += f"&{query_param}"
+
+    try:
+        response = request(
+            method,
+            url,
+            headers=headers
+        )
+        response.raise_for_status()
+    except exceptions.HTTPError as err:
+        print(err)
```

You should probably rethrow the exception, I guess this was just for debugging?